Extension/src/LanguageServer/client.ts

```diff
@@ -262,6 +265,7 @@ class DefaultClient implements Client {
     private failureMessageShown = new PersistentState<boolean>("DefaultClient.failureMessageShown", false);
     private isSupported: boolean = true;
     private inactiveRegionsDecorations = new Map<string, DecorationRangesPair>();
+    private customConfigurationProvider: CustomConfigurationProvider;
```

Comment:
Then will we also need do to some work in configurations.ts to ensure the provider name shows up in the status bar UI instead of the c_cpp_properties.json config.